package.json

```diff
@@ -24,26 +24,26 @@
     },
     "dependencies": {
         "glob": "^7.1.6",
-        "qs": "^6.10.1",
-        "ts-morph": "^20.0.0"
+        "qs": "^6.12.1",
+        "ts-morph": "^22.0.0"
     },
     "devDependencies": {
         "@types/glob": "^7.1.3",
-        "@types/jest": "^29.5.1",
-        "@types/node": "^14.14.35",
-        "@types/qs": "^6.9.6",
+        "@types/jest": "^29.5.12",
+        "@types/node": "^20.12.11",
+        "@types/qs": "^6.9.15",
         "axios": "^0.21.1",
-        "husky": "^4.3.8",
-        "jest": "^29.5.0",
-        "prettier": "^2.2.1",
-        "pretty-quick": "^3.1.0",
+        "husky": "^9.0.11",
```

Comment:
Husky should add a "prepare" script when initialized, according to docs: https://typicode.github.io/husky/get-started.html#husky-init-recommended